ports/esp32/boards/ESP32_S2_WROVER/mpconfigboard.h

```diff
@@ -0,0 +1,12 @@
+#define MICROPY_HW_BOARD_NAME "ESP32-S2-WROVER"
+#define MICROPY_HW_MCU_NAME "ESP32-S2"
+
+#define MICROPY_PY_BLUETOOTH                (0)
+#define MICROPY_HW_ENABLE_SDCARD            (0)
+
+#define MICROPY_HW_I2C0_SCL (9)
+#define MICROPY_HW_I2C0_SDA (8)
```

Comment:
I2C1 has default pins 9 and 8, so with this configuration I2C(0) and I2C(1) will use the same pins.  Does it make sense to configure them differently (eg put I2C(0) on other pins)?

Reply:
Good point I have moved I2C(0).